package.json

```diff
@@ -61,7 +66,6 @@
     "gulp": "^4.0.0",
     "gulp-babel": "^8.0.0",
     "gulp-eslint": "5.0.0",
-    "gulp-util": "^3.0.7",
```

Comment:
Can this also be removed from the dependencies like you did here or is it required?

Reply:
Have done it locally, pushing it now